web/docs/Backend Starters/Decoupled Drupal/creating-new-project.md

```diff
@@ -21,58 +21,26 @@ sidebar_position: 2
 
 ### Installing using Dashboard Upstream
 
-- Create an empty upstream site one of two ways:
+- Create from Decoupled Drupal Composer Managed upstream:
 
   - Via the Pantheon Dashboard at this link:
 
-    - [Empty Upstream](https://dashboard.pantheon.io/sites/create?upstream_id=4c7176de-e079-eed1-154d-44d5a9945b65)
+    - [Decoupled Drupal Composer Managed](https://dashboard.pantheon.io/sites/create?upstream_id=c76c0e51-ad85-41d7-b095-a98a75869760)
 
   - Or Alternatively via Terminus:
 
     ```
-    terminus site:create my-new-site "Describe Site" empty --org='My Team Name'
+    terminus site:create my-new-site "Describe Site" empty --org='My Team Name' c76c0e51-ad85-41d7-b095-a98a75869760
     ```
 
-  - Replace `'{My Team Name}'` with your team name - for example `My Agency`. This can also be omitted.
-
-#### In the Pantheon Admin Dashboard:
-
-Switch **Development Mode** to **Git**
-
-We recommend that you upgrade from the sandbox version of the site plan to one that's appropriate for you using the **Upgrade** button, and change the site plan.
-
-- Clone your site's repository on your local
-
-- Change into the newly cloned directory
-
-- Add remote for the decoupled upstream: `git remote add upstream https://github.com/pantheon-systems/decoupled-drupal-composer-managed`
-
-- Checkout the main branch from upstream:
-
-```
-
-git fetch upstream main
-
-git checkout -t upstream/main
-
-```
-
-- Reset the master branch to match main, then force push that to the Dev environment:
-
-```
-
-git branch -D master
+  **Note:**
```

Comment:
Could you switch this to use Docusaurus's 'note' callout style - change to `:::note`

Reply:
Should we also update the other note formats?